internal/pkg/manifest/rd_web_svc.go

```diff
@@ -43,8 +43,9 @@ type RequestDrivenWebServiceProps struct {
 
 // AppRunnerInstanceConfig contains the instance configuration properties for an App Runner service.
 type AppRunnerInstanceConfig struct {
-	CPU    *int `yaml:"cpu"`
-	Memory *int `yaml:"memory"`
+	CPU      *int   `yaml:"cpu"`
+	Memory   *int   `yaml:"memory"`
+	Platform string `yaml:"platform,omitempty"`
```

Comment:
I worry that having `Platform` as `string` instead of `*string` would result in it getting overriden with `""` if the environment manifest config doesn't specify `platform` 🤔 . 

If it gets overridden as `""`, that could be a problem for users that are not using `linux/amd64` right? If this indeed is a potential impact, could we double check in `ApplyEnv`'s unit test?

Same with the `Platform` in `TaskConfig`.

Sorry for not seeing this in my previous review!!! 🙏🏼